app/components/groups/add-groups.jade

```diff
@@ -0,0 +1,81 @@
+div.row.new-groups-inp(
+  ng-controller="AddGroupController"
+  ng-init="actions.action='default'"
+  ng-switch="actions.action"
+)
+  div.col-md-6.col-md-offset-3.new-group(
+    ng-switch-when="default"
+    ng-click="actions.action='new-task'"
+  )
+    span New group
+    span.ctl-pane.pull-right
+      i.glyphicon.glyphicon-th-list.ctrl-btn(
+        tooltip-placement="bottom"
+        tooltip="Create new list"
+        ng-click="$event.stopPropagation()"
+        )
+      i.glyphicon.glyphicon-search.ctrl-btn(
+        tooltip-placement="bottom"
+        tooltip="Search"
+        ng-click="actions.action='search'"
+        )
+  div.col-md-6.col-md-offset-3.new-group(
+    ng-switch-when="new-task"
+    )
+    div.task-title(
+      contentEditable="true"
+      ng-model="newGroup.title"
+      placeholder="Input a title"
+      ng-init="view.isList=false"
```

Comment:
I see you use `ngInit` directive often. I think it's not good, since it specified in AngularJS docs https://docs.angularjs.org/api/ng/directive/ngInit The only appropriate use of ngInit is for aliasing special properties of ngRepeat, as seen in the demo below. Besides this case, you should use controllers rather than ngInit to initialize values on a scope.